### ISSUE TITLE
loss functions

### ISSUE THREAD

**Issue by aviviz** [matterport/Mask_RCNN#729]
i am trying to train data, and get poor results while adding a lot of "normals" without masks.
even though the loss looks pretty good. 

i wanted to ask about the loss. 
i saw that in mask loss and bbox loss os written: Only positive ROIs contribute to the loss.

what does it mean?  
is that mean that only what the computer guessed right is in the loss? 
y does it true?

and what will happen if i change it to all, and not onlt positive?
